src/Bundle.js

```diff
@@ -228,6 +227,14 @@ export default class Bundle {
 		return this.intro + body;
 	}
 
+	isEmpty () {
+		if (this.intro.length && this.intro.trim())
+			return false;
+		if (this.sources.some(source => !source.content.isEmpty()))
```

Comment:
Should we avoid `some` in favor of simple `for .. of` loops? They also get transpiled into `for` loops by Buble here.

Reply:
Ahh, I didn't know Buble would handle the transpilation here.

Could do, although at this order of loop it probably isn't much of a difference.